### ISSUE TITLE
Please add reading of /etc/shuttle-go.json config

### ISSUE THREAD

**Issue by marcinn** [abourget/shuttle-go#1]
I'd like to package shuttle-go and provide a default config, to make it runnable without requiring user's attention to finish setup. Package scripts can't touch user's $HOME, so the default config must be delivered somewhere else (and typically this is /etc/). That's why I'm suggesting to read the default config from `/etc/shuttle-go.json` when `$HOME/.shuttle-go.json` does not exists.

**Comment by abourget** [comment]
Good idea, and simple contribution if anyone wants to take a stab a it.. I'll gladly merge (unless I have time to do it myself some time)
